### DISCUSSION TITLE
Vscode howto

### DISCUSSION THREAD

**Discussion by gertcuykens** [pandas-dev/pandas-stubs#357] [Q&A] [answered]
Hi, I have a hard time figuring out how i can tell vscode to use the repo instead of the shipped typeshed
  Tried this, but non seem to work

```json
  "python.autoComplete.extraPaths": [],
  "python.languageServer": "Pylance",
  "python.analysis.extraPaths": [
    // ??? "/Users/gert/pandas-stubs/pandas-stubs/pandas-stubs"
  ],
  "python.analysis.typeshedPaths": [
    // ??? "/Users/gert/pandas-stubs"
  ],
  "python.analysis.stubPath": "typings",
  // ??? "python.analysis.stubPath": "/Users/gert/pandas-stubs/pandas-stubs/pandas-stubs",
  ```
  
  Also is the readme still accurate?

<img width="613" alt="image" src="https://user-images.githubusercontent.com/863731/194024956-60c2a81d-6921-4b9c-ac1d-9717540913ee.png">

**Comment by twoertwein** [comment, marked as answer]
> Hi, I have a hard time figuring out how i can tell vscode to use the repo instead of the shipped typeshed Tried this, but non seem to work

I believe VSCode gets its stub from here https://github.com/microsoft/python-type-stubs/#upstreamed-libraries where they previously had their own pandas-stubs - but they should now be using the stubs from here. I don't use VScode, but I expect that you might need to update VSCode so that it automatically includes the stubs from here.

> Also is the readme still accurate?

No, it isn't. You can installed the pypi version with `pip install pandas-stubs`

**Reply by gertcuykens** [reply]
Ok working thanks, had to do
`pip3.10 install -U poetry`
`pip3.10 install -e /Users/gert/pandas-stubs`
And remove all custom vscode configuration that had to do with stubs
